recipes/r-rsvg/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = '1.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rsvg
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: rsvg_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/rsvg_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/rsvg/rsvg_{{ version }}.tar.gz
+  sha256: 8ec457f1d52894619329650c58e2597b168e3312a1129d7863e23abad7d54e9e
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
```

Comment:
I think R 3.3. is not compatible with the current pinning of librsvg. `>=3.4.1` will help I think.

Reply:
Unfortunately it looks like it isn't compatible with either R 3.3.2 or R 3.4.1:

```
$ conda build --R 3.4.1 recipes/r-rsvg/
BUILD START: r-rsvg-1.1-r3.4.1_0
    (actual version deferred until further download or env creation)
updating index in: /opt/conda/conda-bld/linux-64
updating index in: /opt/conda/conda-bld/noarch
Can't build /root/staged-recipes/recipes/r-rsvg due to unsatisfiable dependencies:
The following specifications were found to be in conflict:
  - librsvg
  - r-base 3.4.1*
Use "conda info <package>" to see the dependencies for each package.

$ conda build --R 3.3.2 recipes/r-rsvg/
BUILD START: r-rsvg-1.1-r3.3.2_0
    (actual version deferred until further download or env creation)
Can't build /root/staged-recipes/recipes/r-rsvg due to unsatisfiable dependencies:
The following specifications were found to be in conflict:
  - librsvg
  - r-base 3.3.2*
Use "conda info <package>" to see the dependencies for each package.
```

Reply:
maybe this will fix it? https://github.com/conda-forge/r-base-feedstock/pull/27

Reply:
Just looking at the direct dependencies from `conda info` wasn't helpful. They both share `libxml2` and `pango`, but the pinning is the same. So it must be a downstream dependency that conflicts.

Reply:
Yep, @bgruening I think you are right. The pinning for r-base needs to be updated. For example, `r-base` is pinned to `zlib 1.2.8`, which is incompatible with `librsvg`:

```
$ conda create -y -n zlib librsvg zlib==1.2.8
Fetching package metadata .............
Solving package specifications: .

UnsatisfiableError: The following specifications were found to be in conflict:
  - librsvg -> glib 2.55.* -> zlib 1.2.11
  - zlib ==1.2.8
Use "conda info <package>" to see the dependencies for each package.
```

Reply:
Wow! I would have needed much more time to pin that down so quickly! Thanks a lot!
I assume we put this on hold and trigger a rebuild when `r-base` is pinned to a more recent `zlib`?

Reply:
> trigger a rebuild when r-base is pinned to a more recent zlib

@jenzopr Exactly! The pins for R 3.4.1 were just updated in https://github.com/conda-forge/r-base-feedstock/pull/27. So now you'll need to wait until the pins are updated for R 3.3.2.

Reply:
Updated pins for R 3.3.2 are pending in https://github.com/conda-forge/r-base-feedstock/pull/28

Reply:
PR merged, lets wait a few minutes and restart this one here :)